makefile.shade

```diff
@@ -1064,6 +1063,10 @@ macro name='DoRestore'
     }
 
 functions @{
+    private static readonly string[] TPA_IGNORED_ASSEMBLIES = new [] {
```

Comment:
This has to go here because it is referenced from a function. The `var`s at the top are scoped to a specific method in the generated class and thus not available to the other methods.